File: C++/NexWeb_decimal_to_binary.cpp
    #include <iostream>  
    
    int main()  
    {  
    int a[10], n, i;    
    std::cout<<"Enter the decimal number to convert to binary: ";    
    std::cin >> n;    
    for(i=0; n>0; ++i)    
    {    
    a[i]=n%2;    
    n= n/2;  
    }    
    std::cout<<"Binary of the given number = ";    
    for(i=i-1;i>=0 ;--i)    
    {    
    std::cout<<a[i];    
    }    
    return 0;
    }  

File: Go/NexWeb_decimal_to_binary.go
package main

import "fmt"

var n, input int

func main() {
	fmt.Println("Enter a decimal number to get the binary result:")
	fmt.Scanln(&input)

	n = 1

	for n*2 <= input {
		n = n * 2
	}
	fmt.Println("The result is :")
	for n >= 1 {

		fmt.Print(input / n)
		input = input % n
		n = n / 2
	}
}
